packages/@sfx/cache-driver-plugin/test/unit/common/cache-driver-plugin.test.ts

```diff
@@ -0,0 +1,79 @@
+import { CACHE_REQUEST } from '@sfx/events';
+import { expect, sinon, spy, stub } from '../../utils';
+import CacheDriverPlugin from '../../../src/cache-driver-plugin';
+
+describe('CacheDriverPlugin', () => {
+  let cacheDriverPlugin;
+
+  beforeEach(() => {
+    cacheDriverPlugin = new CacheDriverPlugin();
+  });
+
+  describe('metadata getter', () => {
+    it('should have the name `cache_driver`', () => {
+      expect(cacheDriverPlugin.metadata.name).to.equal('cache_driver');
+    });
+
+    it('should specify `cache` and `dom_events` as dependencies', () => {
+      expect(cacheDriverPlugin.metadata.depends).to.have.members(['cache', 'dom_events']);
+    });
+  });
+
+  describe('register()', () => {
+    it('should save the plugin registry for future use', () => {
+      const registry: any = { a: 1, b: 2 };
+
+      cacheDriverPlugin.register(registry);
+
+      expect(cacheDriverPlugin.core).to.equal(registry);
+    });
+  });
+
+  describe('ready()', () => {
+    it(`should listen for ${CACHE_REQUEST}`, () => {
```

Comment:
I'm not sure I'm sold on using dynamic test descriptions, as this adds an extra level of indirection and makes descriptions harder to search. The developer does not even need to know the actual name of the event.